config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :messages, only: [:index]
+  
+  root "root#index"
```

Comment:
- [ ] When I try to get a message from the backend, I get the following error from the backend and I didn't get any response in the frontend, so kindly fix this issue and make sure you get a random message in the frontend app.
![image](https://user-images.githubusercontent.com/53402600/197800476-a3417dd3-b2f6-4f17-9f57-50de25858518.png)